libs/com-getbase-floatingactionbutton-1-10-1-exploded-aar/AndroidManifest.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest
+    package="com.getbase.floatingactionbutton"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:versionCode="15"
+    android:versionName="1.10.1" >
+
+    <uses-sdk
+        android:minSdkVersion="14"
+        android:targetSdkVersion="22" />
```

Comment:
Yes, exactly. That is why I left it untouched. Still hope Google will provide a speed dial fab at some point... :)